levelkdev/swapr-dapp
# [Feature] Redesign liquidity page

# Summary

Fixes #886

This is a collaboration work between me and @Diogomartf. 

This PR redesigns and improves the liquidity and pair pages. Besides that, updates font and background for the whole dapp.

It's worth to mention this redesign isn't mobile ready yet.

![image](https://user-images.githubusercontent.com/23079677/169522373-4c0b13fa-f4dd-4f06-b34b-c7fa4e606a3a.png)

  # To Test

1. Open the page `liquidity`
- [ ] Verify it has a different design
- [ ] Verify that has a "sort by" option with TVL by default
2. Click in "sort by" and choose a new option
- [ ] Verify values are sort by another order
3. Click in a pair
- [ ] Verify it has a different design
- [ ] Verify it fetches TVL, 24H Volume, APY
- [ ] Verify it fetches liquidity position details if you add liquidity
- [ ] Verify there is a new campaign section with pair campaigns
4. Click in "trade" button inside pair
- [ ] Goes to swap page with the selected pair as the trade
5. Click in the "campaign" button inside pair
- [ ] Goes to reward page with the selected pair
6. Click in a random campaign inside pair
- [ ] Goes to selected campaign
